code/__HELPERS/_debugger.dm

```diff
@@ -23,3 +22,13 @@
 	var/debug_server = world.GetConfig("env", "AUXTOOLS_DEBUG_DLL")
 	if (debug_server)
 		call(debug_server, "auxtools_shutdown")()
+
+/// This is here so that the debugger is initialized before (mostly) everything else.
+/// Makes it possible to set breakpoints in the New() of things assigned to
+/// global variables or objects included in a compiled map file.
+/datum/debugger
+
+/datum/debugger/New()
+	world.enable_auxtools_debugger()
+
+GLOBAL_REAL(Debugger, /datum/debugger) = new
```

Comment:
Could you refactor this into `/datum/super_early_init` and also merge the code from `code\__DEFINES\_profile.dm` into it?